addons/web/static/src/views/form/form_controller.scss

```
@@ -668,6 +668,20 @@
         }
     }
 
+    .o_field_float.oe_inline, .o_field_monetary.oe_inline {
+        max-width: 9rem;
+        & input {
+            max-width: 9rem
+        }
```

Avoid duplicated rule :
```suggestion
        &, input {
            max-width: 9rem;
        }
```